code/modules/cargo/packs/engineering.dm

```diff
@@ -12,7 +12,7 @@
 
 /datum/supply_pack/engineering/shieldgen
 	name = "Anti-breach Shield Projector Crate"
-	desc = "Hull breaches again? Say no more with the Nanotrasen Anti-Breach Shield Projector! Uses forcefield technology to keep the air in, and the space out. Contains two shield projectors."
+	desc = "Hull breaches again? Say no more with the Horizons Anti-Breach Shield Projector! Uses forcefield technology to keep the air in, and the space out. Contains two shield projectors."
```

Comment:
Hull breaches again? Say no more with the AccuTech Anti-Breach Shield Projector! Uses forcefield technology to keep the air in, and the space out. Contains two shield projectors.